build.assets/Makefile

```diff
@@ -43,7 +43,7 @@ GID := 1000
 NOROOT := -u 1000:1000
 # if running in CI and the GOCACHE environment variable is not set, set it to a sensible default
 ifeq ("$(GOCACHE)",)
-GOCACHE := $(shell go env GOCACHE)
+GOCACHE := /go/cache
```

Comment:
nit: let's rename `GOCACHE` to `DOCKER_GOCACHE` here to avoid any confusion with the actual `GOCACHE` used by the go tool locally

Reply:
This needs to actually be called `GOCACHE` to set the environment variable correctly though - the intention is just to catch situations where it isn’t set as an environment variable in the Drone pipelines. We could just remove it to make the GOCACHE requirement explicit, but I’d have to test all the pipelines again to make sure they still work.